samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
In K, we aren't really using IsInRole anymore, our AuthZ policies will look for the presence of any claim with type ClaimsTypeRole and the value of the role name from a union claims from a specified set of authentication schemes.